#error more params in func call

def main(a):
    print('in main')
    print(a)

main('1',2)
